include/iso15118/session/feedback.hpp

```diff
@@ -50,6 +59,11 @@ struct Callbacks {
     std::function<void(const message_20::Type&)> v2g_message;
     std::function<void(const std::string&)> evccid;
     std::function<void(const std::string&)> selected_protocol;
+
+    std::function<void(const dt::ServiceCategory&, const dt::AcConnector&, const dt::ControlMode&,
```

Comment:
Why is `dt::AcConnector` not optional? Running DC we dont care if its single or three phase